components/console-backend-service/OWNERS

```diff
@@ -0,0 +1,11 @@
+reviewers:
+  - console-approvers
+  - michal-hudy 
+  - m00g3n
+  - aerfio 
```

Comment:
I have made max 2 PRs in CBS in total, so there's no sense for me to be here, I wouldn't be able to help 😢 
Please remove me too

Reply:
done